python_modules/libraries/dagster-aws/dagster_aws/pipes.py

```diff
@@ -9,12 +9,22 @@
 from dagster._core.pipes.client import (
     PipesParams,
 )
-from dagster._core.pipes.utils import PipesBlobStoreMessageReader
+from dagster._core.pipes.utils import PipesBlobStoreMessageReader, PipesBlobStoreStdioReader
 
 
 class PipesS3MessageReader(PipesBlobStoreMessageReader):
-    def __init__(self, *, interval: float = 10, bucket: str, client: boto3.client):
-        super().__init__(interval=interval)
+    def __init__(
```

Comment:
We should add a docblock, since this just got complicated. In particular explaining the behavior if the readers are `None`

Reply:
Added a docstring here, and modified the docstring of the `PipesBlobStoreMessageReader` base class with a description of the behavior for null readers.

Reply:
I'd just repeat that information here